ViAn/GUI/videowidget.cpp

```diff
@@ -993,6 +1002,11 @@ void VideoWidget::on_video_info(int video_width, int video_height, int frame_rat
     playback_slider->setMaximum(last_frame);
     set_total_time((last_frame + 1) / frame_rate);
     set_current_time(frame_index.load() / m_frame_rate);
+
+    video_loaded.store(true);
```

Comment:
Is this the same atomic as in videoplayer (m_video_loaded)? If so it shouldn't be set here? Both video_loaded and new_frame should be updated by the videoplayer when it actually has something new otherwise it will probably happen in the wrong thread

Reply:
Yes. This was among the things i tried. Since I don't want these variables to be set before the video is actually loaded and all variable from that is set. But it doesn't seems to be necessary.